src/org/usfirst/frc4904/robot/humaninterface/HumanInterfaceConfig.java

```diff
@@ -0,0 +1,12 @@
+package org.usfirst.frc4904.robot.humaninterface;
+
+import org.usfirst.frc4904.robot.vision.AligningCameraGRIP;
+import org.usfirst.frc4904.sovereignty.strategies.GearAlign;
+
+public class HumanInterfaceConfig {
+
```

Comment:
Please remove this blank line, it's against convention but the formatter can't remove it